_15thnight/__init__.py

```diff
@@ -53,194 +60,40 @@ def shutdown_session(response):
     database.db_session.remove()
 
 
-@app.errorhandler(404)
-@app.errorhandler(Exception)
-def error_page(error):
-    """Generic Error handling."""
-    code = 500
-    if isinstance(error, HTTPException):
-        code = error.code
-
-    print(error)
-    return render_template("error.html", error_code=code), code
+if not app.config['DEBUG']:
```

Comment:
This should probably be:

    if not app.config.get("DEBUG"):

in case the option is missing from the config.